tests/executables_src/testBidirectionalVariables.cc

```diff
@@ -544,16 +544,20 @@ BOOST_AUTO_TEST_CASE(testDataValidityReturn) {
     app.lower.incrementDataFaultCounter();
     app.lower.var = 120;
     app.lower.var.write();
+    app.lower.decrementDataFaultCounter();
```

Comment:
Just for clarification, I would move this line down below the two corresponding BOOST_CHECK lines, as it is not part of this test really but rather just restores the state as before the test.